cypress/support/step_definition/course/editCourse/editCourse.spec.js

```diff
@@ -0,0 +1,52 @@
+import Edit_Course from '../../../pages/course/pageEditCourse'
+import { And, When } from 'cypress-cucumber-preprocessor/steps'
+
+const editCourse = new Edit_Course()
+
+And('I click on course', () => {
```

Comment:
See above, already exists in deleteCourse.spec.js.
We should keep one, and maybe move it to a more gereral area and remove the other.

Reply:
As d3n153 created a pageCommon.js in the branch added new school administration, these changes should be done after merging both branches.